### DISCUSSION TITLE
Spring Boot 3.2 Support

### DISCUSSION THREAD

**Discussion by andrewkruse** [MarcGiffing/bucket4j-spring-boot-starter#231] [Q&A] [answered]
Spring boot 3.2 is out (and has a feature I want relating to paging...) 

I'd rather not pick and choose, but I'm curious is there any reason that a 0.10.x release would not be compatible with 3.2? I'd rather not fish through dependencies if there is a version for 3.2 that I'm missing or if there's a release coming through in the pipeline somewhere.

https://spring.io/projects/spring-boot#support

I did find this which suggests it is at least a bit compatible, but I never saw a resolution. With the stack trace in question I expect it to be more of a configuration problem than a real compatibility problem though. https://github.com/MarcGiffing/bucket4j-spring-boot-starter/discussions/227

**Comment by MarcGiffing** [comment, marked as answer]
Please use at version 0.12.x or 0.11.x. 

The Version used in #227 is quite old und uses the unsupported redis starter.
